WebRoot/css/dwt.css

```diff
@@ -2041,7 +2039,9 @@ INPUT.xform_field {
 .ZSelectAutoSizingContainer					{	height:2rem; width:auto;	}		/* TODO: SIZE ASSUMPTION! */
 
 .ZSelect									{	@Select@		@SelectSpacing@			}
-.ZSelect .ZWidgetTitle						{	padding-left:5px; text-align:left;		}
+.ZSelect .ZWidgetTitle						{	@SelectTitle@		}
```

Comment:
Yes we used tabs in webclient, so it is creating these issues, if we use spaces then this will be resolved
we can have a separate commit for converting all tabs to spaces atleast in base4 directory